tests/docker/config/construct-query.sparql

```diff
@@ -657,4 +710,115 @@ CONSTRUCT {
                                                   OPTIONAL { ?preSynaptic     skos:notation   ?preSynapticNotation    . } .
 
     } .
+
+  # Morphology features
+  OPTIONAL {
```

Comment:
This is only for neuron morphologies or a subset of types right ?
If yes, we could add something like:
```
?id  a           nsg:NeuronMorphology ;
```
to act as a short circuit so that the query is faster for the other types ?

Reply:
Yes, this is a good idea.